pkgs/top-level/all-packages.nix

```diff
@@ -7029,6 +7039,7 @@ in
   );
 
   libgit2_0_21 = callPackage ../development/libraries/git2/0.21.nix { };
```

Comment:
Did you try to remove this (and upgrade all packages). Assuming semantic versioning the new version should be compatible.